install/kubernetes/tetragon/values.yaml

```diff
@@ -269,14 +269,27 @@ tetragonOperator:
     create: true
     annotations: {}
     name: ""
-  # -- securityContext for the Tetragon Operator Deployment Pods.
+  # -- securityContext for the Tetragon Operator Deployment Pods. (commented out for backwards compatability)
   podSecurityContext: {}
-  # -- securityContext for the Tetragon Operator Deployment Pod container.
-  securityContext:
+  #  runAsNonRoot: true
+  #  runAsUser: 65532
+  #  runAsGroup: 65532
+  #  fsGroup: 65532
+  #  seccompProfile:
+  #    type: RuntimeDefault
+  # -- securityContext for the Tetragon Operator Deployment Pod container. (commented out for backwards compatability)
+  containerSecurityContext:
```

Comment:
so I'm not against deprecating `operator.securityContext` and use `operator.containerSecurityContext` so that it's more explicit but could you move the deprecation in a separate earlier commit and make the fallback natural: meaning that if `containerSecurityContext` is not defined, templating should still use `securityContext` so that it's backward compatible? Thanks